### ISSUE TITLE
What is the best approach for Summary links

### ISSUE THREAD

**Issue by gep13** [cake-build/cake#491]
When updating the GitVersion documentation, I did this:

![image](https://cloud.githubusercontent.com/assets/1271146/10969185/93a6d3fa-83be-11e5-8cd8-c4009fdf64e2.png)

Thinking it would me line breaks between them, but I think I need to be more explicit, as the end result is this:

![image](https://cloud.githubusercontent.com/assets/1271146/10969156/65b3b49a-83be-11e5-8cd1-3ee37627bf91.png)

I think we need to decide on a standard format for the summary sections of each alias, and apply them throughout.

Thoughts?

**Comment by devlead** [comment]
As I think @patriksvensson has suggested before the one way would be to support the [list](https://msdn.microsoft.com/en-us/library/y3ww3c7e.aspx) tag,

The way I've seen in standard XML docs is using the [para](https://msdn.microsoft.com/en-us/library/x640hcd2.aspx) tag, don't know if we support that yet though.
```csharp
    /// <summary>DoWork is a method in the TestClass class.
    /// <para>Here's how you could make a second paragraph in a description. <see cref="System.Console.WriteLine(System.String)"/> for information about output statements.</para>
    /// <seealso cref="TestClass.Main"/>
    /// </summary>
    public static void DoWork(int Int1)
    {
    }
```

**Comment by patriksvensson** [comment]
@devlead @gep13 The `para` tag is supported.

Here is a [list of what is supported by the documentation tool](https://github.com/cake-build/website/tree/develop/src/Cake.Web.Docs/Xml/Parsers) at the moment. I think the only thing that is missing is lists. I can make a stab tonight to implement that as well.

**Comment by patriksvensson** [comment]
And there's also an implementation of `para` rendering: https://github.com/cake-build/website/blob/develop/src/Cake.Web.Core/Rendering/CommentRenderer.cs#L85

**Comment by devlead** [comment]
@patriksvensson then `para` should work just fine for @gep13 ;)

About `lists` think [highlight.js](https://highlightjs.org/) is more important so we can get [contributing/documentation](http://cakebuild.net/docs/contributing/documentation) looking ok ;)
Added issue cake-build/website#23 for this.

I will add supported tags to [contributing/documentation](http://cakebuild.net/docs/contributing/documentation) so we have that documented too.
Added issue cake-build/website#24 for this.

**Comment by gep13** [comment]
This also looks like a duplicate of: https://github.com/cake-build/cake/issues/150 so going to close it.

**Comment by gep13** [comment]
This was actually included in the v0.11.0 release, and can be closed.
